beacon-chain/rpc/eth/validator/validator_test.go

```diff
@@ -398,6 +434,19 @@ func TestGetSyncCommitteeDuties_SyncNotReady(t *testing.T) {
 	assert.ErrorContains(t, "Syncing to latest head, not ready to respond", err)
 }
 
+func TestSyncCommitteeDutiesLastValidEpoch(t *testing.T) {
```

Comment:
I don't particularly like this test because it duplicates a part of the implementation. But because I fetch a config value, which changes between regular and minimal configs, I cannot simply hardcode a value here.